code/modules/mob/living/simple_animal/hostile/necro.dm

```diff
@@ -143,53 +143,60 @@
 	Otherwise, start wandering and bust down some doors to find more food
 	*/
 	if(!isUnconscious())
-		if(stance == HOSTILE_STANCE_IDLE) //Not doing anything at the time
+		if(stance == HOSTILE_STANCE_IDLE && (!client)) //Not doing anything at the time
```

Comment:
But everyone knows more parentheses means more safety
<sub>That's an actual line comment in my Steambot's code</sub>